databricks_cli/sdk/api_client.py

```diff
@@ -129,6 +130,27 @@ def perform_query(self, method, path, data = {}, headers = None):
             raise requests.exceptions.HTTPError(message, response=e.response)
         return resp.json()
 
+    @classmethod
+    def enable_debug_logging(cls):
+        # These two lines enable debugging at httplib level (requests->urllib3->http.client)
+        # You will see the REQUEST, including HEADERS and DATA, and RESPONSE with HEADERS but without DATA.
+        # The only thing missing will be the response.body which is not logged.
+        try:
+            import http.client as http_client
+        except ImportError:
+            # Python 2
+            import httplib as http_client
+
+        print ("HTTP debugging enabled")
+        http_client.HTTPConnection.debuglevel = 1
+
+        # You must initialize logging, otherwise you'll not see debug output.
+        logging.basicConfig()
```

Comment:
What do you think about putting the logic here? https://github.com/databricks/databricks-cli/blob/master/databricks_cli/click_types.py#L104.

Reply:
Let me move it there, that is a better place.  This set of patches is older, so I did have the same understanding I do now.

Reply:
Moved thank you.